### PR TITLE
[v0.0.2] Update docs, fix imports, fix unpkg links, move data files

### PR DESCRIPTION
**Pull request by xhlulu**
Will bump version when changes are accepted.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -18,6 +18,8 @@
     license=package['license'],
     description=package['description'] if 'description' in package else package_name,
     install_requires=[
-        'colour==0.1.5'
+        'dash',
+        'dash_html_components',
+        'dash_renderer',
```
Comment (chriddyp) [inline setup.py]: I think that this should technically be `dash-html-components` and `dash-renderer`

Comment (xhlulu) [reply setup.py]: Yes, good catch! Updating this. now

Comment (xhlulu) [reply setup.py]: I think the reason for this error is that I copied directly from dash's install_requires:
https://github.com/plotly/dash/blob/3533073d1e2de5ad208a278d604822afbb9c57fa/setup.py#L19-L23

Looking at other `setup.py` examples, `dash-html-components` is indeed the right way to do it.

**Comment by xhlulu** [comment]
Interestingly, both `dash_renderer` and `dash-renderer` would work, since they are recognized as the same package by pip. In fact, it would also work for `Flask` and `flask`:

```commandline
C:\Users\xingh>pip show Flask
Name: Flask
Version: 1.0.1
Summary: A simple framework for building complex web applications.
Home-page: https://www.palletsprojects.com/p/flask/
Author: Armin Ronacher
Author-email: armin.ronacher@active-4.com
License: BSD
Location: c:\users\xingh\appdata\local\continuum\anaconda3\lib\site-packages
Requires: Jinja2, Werkzeug, itsdangerous, click
Required-by: Flask-Cors, Flask-Compress, Flask-Caching, dash
You are using pip version 18.0, however version 18.1 is available.
You should consider upgrading via the 'python -m pip install --upgrade pip' command.

C:\Users\xingh>pip show flask
Name: Flask
Version: 1.0.1
Summary: A simple framework for building complex web applications.
Home-page: https://www.palletsprojects.com/p/flask/
Author: Armin Ronacher
Author-email: armin.ronacher@active-4.com
License: BSD
Location: c:\users\xingh\appdata\local\continuum\anaconda3\lib\site-packages
Requires: Werkzeug, click, Jinja2, itsdangerous
Required-by: Flask-Cors, Flask-Compress, Flask-Caching, dash
You are using pip version 18.0, however version 18.1 is available.
You should consider upgrading via the 'python -m pip install --upgrade pip' command.
```

On the other hand, it wouldn't work for a package like `scikit-learn` (i.e. `pip show sklearn` doesn't print anything). From what I observed (e.g. Flask) it is more common to include the PyPi distribution of the package, rather than the repo name:

dash_renderer: https://pypi.org/project/dash_renderer/
dash-html-components: https://pypi.org/project/dash-html-components/

So in this case it would be `dash_renderer` and `dash-html-components`. However, I believe this is not strictly enforced, since `-` are directly converted to `_`, and vice versa. Similarly for lower case and upper case.

In fact `pip show DASH-HtMl_comPONENTS` would yield the same results as `pip show dash-html-components`, and https://pypi.org/project/DASH-HtMl_comPONENTS would redirect to https://pypi.org/project/dash-html-components/.

**Pull request merged**